### ISSUE TITLE
Default connection for Doctrine is hard-coded in

### ISSUE THREAD

**Issue by natelenart** [ldaptools/ldaptools-bundle#6]
Hello, I'm looking to upgrade my current LDAP implementation to this bundle, which looks way better and more complete than what I'm using.  I got partway into switching, then hit an error:

```
 [RuntimeException]
  The Doctrine connection "default" referenced in service "ldap_tools.doctrine.event_listener.ldap_object" does not exist. Available connections names: ...
```

My project currently doesn't have a Doctrine connection named "default".  I looked through your code trying to see if I could configure it somehow, but it looks like it's hardcoded into the `addTag` call:

```
$container->getDefinition('ldap_tools.doctrine.event_listener.ldap_object')->addTag(
    'doctrine.event_subscriber',
    ['connection' => 'default']
);
```

Is this something that could be changed to be configurable?  I realize my situation isn't common (not having a "default" connection, but at the same time, it's not unreasonable to think that other users might have multiple connections and the one they might want to use might not be the default one.

**Comment by ChadSikorra** [comment]
Yeah, it's definitely something I should change. It seems I probably read the docs wrong when I was looking at the examples:

http://symfony.com/doc/current/doctrine/event_listeners_subscribers.html

It clearly says if you don't specify a connection it will subscribe to all of them. So I shouldn't really even specify the connection, which would fix the issue. I'll also make it configurable option in case someone wants to only use it of a specific connection. Should be able to get around to this pretty soon.

**Comment by ChadSikorra** [comment]
I removed the hard-coding in this commit: https://github.com/ldaptools/ldaptools-bundle/commit/460d3f0eb7b24a79ee83ebbdca3ff5319ee6f516. It will now register against all Doctrine connections by default. I also added options to disable the integration altogether or only enable the integration for specific Doctrine connections. I added a configuration reference and documented the options here: https://github.com/ldaptools/ldaptools-bundle/blob/master/Resources/doc/Configuration-Reference.md#doctrine

Thanks again for reporting this! I will likely tag another version in a few weeks. I'd like to release the next version of LdapTools then bump the version in this bundle first.
